cves/CVE-2017-15707.yml

```diff
@@ -205,4 +232,19 @@ mistakes:
     Use those questions to inspire your answer. Don't feel obligated to answer
     every one. Write a thoughtful entry here that those in the software
     engineering industry would find interesting.
-  answer:
+  answer: |
+    The main mistake that was made was keeping around old, outdated libraries
+    and still using them, despite them not receiving any security updates. The
+    team began using this library in 2007 and didn't stop using it until it was
+    discovered that a user could cause a DOS attack.
+
+    The team seems to have taken the proper initiatve of replacing the old
+    library with a new one (Jackson), that is still being worked on and
+    receiving security updates. It did take some time for them to switch things
+    over to using Jackson though as can be seen in some later commits showing
+    them switching to Jackson up to two months after the initial switch was
+    made. It was also noted that instead of fully removing the ability to use
+    the outdated json-lib library, they instead leave it as an optional thing,
+    with Jackson acting as the defualt JSON reader. They seem to making proper
+    steps towards preventing the vulnerability, but still have some work to do,
+    as there was no mention of removing json-lib as optional.
```

Comment:
Maybe you can write out your thoughts on why they are keeping the json-lib as an option when it is clearly at risk of having multiple security vulnerabilities. Although Jackson was set as the default setting, I'm sure there are some users that may still opt to use json-lib without knowing full well why they switched over to Jackson.